### DISCUSSION TITLE
Reposync bandwith control

### DISCUSSION THREAD

**Discussion by dfaltum** [uyuni-project/uyuni#9390] [Ideas]
Hello there!

Our customer has a problem when SUSE Manager synchronizes a new repo. The entire company's bandwidth is throttled because of this process. They can't manage the network devices. 

We could use a solution like tc or wondershaper for bandwith control, but I don't think that's a good idea. With these solutions we would also be limiting the salt and the internal HTTP traffic. Maybe we can configure an HTTP proxy somewhere (or a podman container on the SUMA host itself). Do you have any suggestions for a problem like this?

**Comment by mcalmer** [comment]
By default reposync download with 5 threads in parallel. Maybe it helps already when you limit this to 1.
Modify /etc/rhn/rhn.conf and add

```
server.satellite.reposync_download_threads = 1
```

**Reply by dfaltum** [reply]
Thanks for the idea! It helped a bit, but still not ideal.
